salt/metalk8s/defaults.yaml

```diff
@@ -105,4 +105,11 @@ coredns:
 
 upgrade: false        # define if we're on an upgrade case
 
+# Define if the downgrade from this version is supported.
+# It should be set to false if manual operations are needed
+# (e.g. downgrade of etcd), prior to downgrading the cluster.
+# The downgrade can still be forced setting force_downgrade to True
+# in the pillar.
```

Comment:
Good idea to make this overridable, but why not use the same name in both defaults and pillar, and merge them in `map.jinja`?